x-pack/test/cloud_security_posture_functional/cloud_tests/agentless_sanity.ts

```diff
@@ -5,94 +5,131 @@
  * 2.0.
  */
 
-import { CLOUD_CREDENTIALS_PACKAGE_VERSION } from '@kbn/cloud-security-posture-plugin/common/constants';
 import expect from '@kbn/expect';
 import type { FtrProviderContext } from '../ftr_provider_context';
 // eslint-disable-next-line import/no-default-export
 export default function ({ getPageObjects, getService }: FtrProviderContext) {
+  const testSubjects = getService('testSubjects');
+  const find = getService('find');
+  const queryBar = getService('queryBar');
   const pageObjects = getPageObjects([
     'common',
     'cspSecurity',
     'security',
     'header',
     'cisAddIntegration',
+    'findings',
+    'fleet',
   ]);
 
-  const CIS_AWS_OPTION_TEST_ID = 'cisAwsTestId';
+  describe('Agentless Cloud - Sanity Tests', function () {
+    describe('agentless agent health', function () {
```

Comment:
I'm not sure if testing agent health through the UI is the most suitable approach for our case. The Fleet component isn't under our responsibility, and since you're not using any specific test subjects for object identification, there's a risk that if the Fleet team changes something related to the UI, it could break our tests.

If we still want to pinpoint where our tests are failing—whether the issue is on the UI side or due to the agent's health—it might be better to use a [Fleet API call](https://www.elastic.co/guide/en/fleet/master/fleet-apis.html#getAgents) to verify if the required agent is healthy. 

In the next test suite, we are already querying agentless data. If that suite fails, it should be sufficient to help identify where the problem lies.

Reply:
I've since removed the health checks and decided to simplify this and only check that we receive findings.  If we are receiving findings that should indicate a good health and working Agentless agent.